core/modules/admin_bar/css/admin_bar.css

```diff
@@ -229,15 +239,15 @@
   background: #ffffff;
   width: 1em;
   height: 100%;
-  margin: -0.75em 0;
+  margin: -0.5em 0;
   position: absolute;
-  right: 1em; /* LTR */
+  right: .7em; /* LTR */
```

Comment:
Add a zero before the point for consistency? (`0.7em`) See also below, line 250.

Reply:
Good point, let's be consistent. :+1: